sealink/Formerly
# [DC-1475]  Add reCaptcha validation

### Please don't merge it to master yet! I'll merge it when I finished the One SeaLink PR review (https://bitbucket.org/team-sealink/one-sealink/pull-requests/200/dc-1476-add-recaptcha-to-craft-contact/diff)

### Why
WA are getting 1000's of spam emails from their contact form. We need to add reCaptcha to our contact forms. This needs to be off by default with the option to turn it on per form.

### Test
1. Change the  'Formerly' plugin's version on the composer.json (One SeaLink)
`"sealink/formerly": "1.7.0-alpha.0",`
2. Run `composer clear-cache` and `composer update formerly`
3. Run One SeaLink project and go to CMS 
4. Click the 'Formerly' on the left menu bar and select an form
5. Check the 'Use Google reCaptcha'
6. Input the Google keys for reCaptcha
You can check the keys here: https://www.google.com/recaptcha/admin
dsm.sealink@gmail.com account.

### Result
**CMS**
![screen shot 2018-08-02 at 11 51 25 am](https://user-images.githubusercontent.com/26703960/43558807-6d5ed516-964a-11e8-8424-a5ace17e38ff.png)

![screen shot 2018-08-02 at 11 51 56 am](https://user-images.githubusercontent.com/26703960/43558820-7fb8b1be-964a-11e8-9892-299db0feaf57.png)
